app/templates/.jshintrc

```diff
@@ -12,6 +12,7 @@
 	"unused": true,
 	"strict": true,
 	"boss": true,
+	"validthis": true,
```

Comment:
This option suppresses warnings about possible strict violations when the code is running in strict mode and you use this in a non-constructor function.